include/SugarObjects/LanguageManager.php

```diff
@@ -174,6 +174,7 @@ public static function clearLanguageCache($module_dir = '', $lang = '')
      */
     private static function _clearCache($module_dir, $lang = null)
     {
+        if ($module_dir === 'unified_search_modules.php') return;
```

Comment:
There should be a curly brace around the nested statement as per PSR-2.

```suggestion
        if ($module_dir === 'unified_search_modules.php') {
            return;
        }
```